x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -707,13 +721,15 @@ export class TaskRunner<
       schedule = asErr(err);
     }
 
+    const nextRunString = isOk(schedule) ? getNextRunString(schedule.value.interval) : null;
```

Comment:
This looks like it's calculating the next run from the current time + schedule interval. Task manager will calculated it from the `task.startedAt` time + schedule interval. Maybe we should do that as well for consistency.